libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -90,6 +90,12 @@ const AP_Param::GroupInfo AP_RSSI::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("CHAN_HIGH", 6, AP_RSSI, rssi_channel_high_pwm_value,  2000),
 
+    // @Param: SCALE
+    // @DisplayName: Scale factor for rssi signal
+    // @Description: Scale factor for rssi signal
+    // @Range: 0 100
+    // @User: Standard
+    AP_GROUPINFO("SCALE", 7, AP_RSSI, rssi_scale_factor,  1),
```

Comment:
I wonder if we should default to `-1` meaning, "default value" - which we might be able to heuristically determine based on the numbers we're receiving.  Or potentially "work it out" which overwrites the parameter value if we can work it out based on the rssi we get from the receiver.